modules/programs/fish.nix

```
@@ -8,6 +8,10 @@ let
 
   cfg = config.programs.fish;
 
+  fishAbbrs = concatStringsSep "\n" (
+    mapAttrsToList (k: v: "abbr -ag ${k} ${escapeShellArg v}") cfg.shellAbbrs
```

```suggestion
    mapAttrsToList (k: v: "abbr -a ${k} -- ${escapeShellArg v}") cfg.shellAbbrs
```
The `-g` flag does not exist in 4.0 and I think it was removed in 3.6 or earlier. The `--` will allow defining expansions that start with `-`